presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/PinotColumnMetadata.java

```diff
@@ -30,7 +29,7 @@ public class PinotColumnMetadata
 
     public PinotColumnMetadata(String name, Type type, boolean inNullable, String comment)
     {
-        super(requireNonNull(name, "name is null"), requireNonNull(type, "type is null"), inNullable, comment, null, false, ImmutableMap.of());
+        super(name, type, inNullable, comment, null, false, ImmutableMap.of());
```

Comment:
Why is this changed? and is there a reason we removed the `requireNonNull`?

Reply:
So we already have these checks in ColumnMetadata here https://github.com/prestodb/presto/pull/24616/files#diff-11c55ceab33883d51ac1741344d33b77c6737942ac534c409a6c73af2d4231d0R46
So I have taken it out from here.